src/Approximations/box_approximation.jl

```diff
@@ -331,3 +325,75 @@ end
 function box_approximation(B::Union{Ball1, Ball2, BallInf, Ballp})
     return Hyperrectangle(center(B), fill(B.radius, dim(B)))
 end
+
+"""
+    box_approximation(ch::ConvexHull; [algorithm]::String="lowhigh")
+
+Overapproximate a convex hull with a tight hyperrectangle.
+
+### Input
+
+- `ch`        -- convex hull
+- `algorithm` -- (optional; default: `"lowhigh"`) algorithm choice
+
+### Output
+
+A hyperrectangle.
+
+### Algorithm
+
+Let `X` and `Y` be the two sets of `ch`.
+
+If `algorithm == "lowhigh"`, we compute the low and high coordinates of `X` and
+`Y` via `low_high`.
+
+If `algorithm == "twobox"`, we instead compute the box approximations of `X` and
+`Y` via `box_approximation`.
+
+In both cases we then take the interval hull.
+The `"lowhigh"` algorithm is more efficient if `low_high` is efficient because
+it does not need to allocate the intermediate hyperrectangles.
+"""
+function box_approximation(ch::ConvexHull; algorithm::String="lowhigh")
+    if algorithm == "lowhigh"
+        return _box_approximation_chull_lowhigh(ch.X, ch.Y)
+    elseif algorithm == "twobox"
+        return _box_approximation_chull_twobox(ch.X, ch.Y)
+    else
+        throw(ArgumentError("unknown algorithm $algorithm"))
+    end
+end
+
+function _box_approximation_chull_lowhigh(X::LazySet{N}, Y) where {N}
+    n = dim(X)
+    c = Vector{N}(undef, n)
+    r = Vector{N}(undef, n)
+    for i in 1:n
+        l1, h1 = low_high(X, i)
+        l2, h2 = low_high(Y, i)
+        l = min(l1, l2)
+        h = max(h1, h2)
+        ci = (h + l) / 2
+        c[i] = ci
+        r[i] = h - ci
+    end
+    return Hyperrectangle(c, r)
+end
+
+function _box_approximation_chull_twobox(X, Y)
+    n = dim(X)
+    H1 = box_approximation(X)
+    H2 = box_approximation(Y)
+    l = [min(low(H1, i), low(H2, i)) for i in 1:n]
+    h = [max(high(H1, i), high(H2, i)) for i in 1:n]
+    ci = (h + l) / 2
+    c = ci
+    r = h - ci
+    return Hyperrectangle(c, r)
+end
+
+function box_approximation(ms::MinkowskiSum)
```

Comment:
suggestion to add the docstring for later

````julia
    box_approximation(ms::MinkowskiSum)

Compute the box approximation of the lazy Minkowski sum of sets.

### Input

- `ms` -- 
 
### Output

Hyperrectangle representing the box approximation of `ms`.

### Algorithm

The box approximation distributes over Minkowski sums, i.e.

```math
\\square(X \\oplus Y) = \\square(X) \\oplus \\square(Y)
```
It suffices to compute the box approximation of each element, then take the concrete Minkowski sum of hyperrectangles.
````